src/main/java/com/salesforce/dataloader/ui/entitySelection/EntityFilter.java

```diff
@@ -70,19 +72,20 @@ public boolean select(Viewer arg0, Object arg1, Object arg2) {
                     || entityName.equals("Event") || entityName.equals("Lead") || entityName.equals("Opportunity")
                     || entityName.equals("Pricebook2") || entityName.equals("Product2") || entityName.equals("Task")
                     || entityName.equals("User")) {
-                return filterBySearchText(entityName);
+                return filterBySearchText(entityName, entityLabel);
             } else if (entityName.endsWith("__c")) {
-                return filterBySearchText(entityName);
+                return filterBySearchText(entityName, entityLabel);
             }
             return false;
         }
     }
     
-    private boolean filterBySearchText(String entityName) {
+    private boolean filterBySearchText(String entityName, String entityLabel) {
         String searchText = this.searchText.getText();
         if (searchText != null && !searchText.isEmpty() && entityName != null) {
             searchText = searchText.toLowerCase();
-            if (entityName.toLowerCase().contains(searchText)) {
+            if (entityName.toLowerCase().contains(searchText) 
+                    || entityLabel.toLowerCase().contains(searchText)) {
```

Comment:
Do we also need a null check for entityLabel?

Reply:
Done. please review.

Reply:
There is no need for a null check here, it does not hurt, but its not required. This value comes from the describe result so it cannot be null.